kore/src/Kore/JsonRpc.hs

```diff
@@ -134,7 +136,7 @@ respond serverState moduleName runSMT =
                                             then EnableMovingAverage
                                             else DisableMovingAverage
                                         )
-                                        (fromMaybe False $ liftA2 (||) logSuccessfulRewrites logSuccessfulSimplifications)
```

Comment:
This evaluates to `False` if `logSuccessfulSimplifications` is `Nothing`, independent of `logSuccessfulRewrites`.